examples/helm/kanister/kanister-mysql/kanister/mysql-blueprint.yaml

```diff
@@ -1,68 +1,70 @@
+apiVersion: cr.kanister.io/v1alpha1
+kind: Blueprint
+metadata:
+  name: mysql-blueprint
 actions:
   backup:
     type: Deployment
     outputArtifacts:
       mysqlCloudDump:
         keyValue:
-          path: '/mysql-backups/{{ .Deployment.Namespace }}/{{ .Deployment.Name }}/{{ toDate "2006-01-02T15:04:05.999999999Z07:00" .Time  | date "2006-01-02T15-04-05" }}/dump.sql.gz'
+          path: /mysql-backups/${MYSQL_NAMESPACE}/${MYSQL_RELEASE}-mysql/{{ toDate "2006-01-02T15:04:05.999999999Z07:00"
+            .Time  | date "2006-01-02T15-04-05" }}/dump.sql.gz
     phases:
-    - func: KubeExec
+    - func: KubeTask
       name: dumpToObjectStore
       args:
-        namespace: "{{ .Deployment.Namespace }}"
-        pod: "{{ index .Deployment.Pods 0 }}"
-        container: kanister-sidecar
+        image: kanisterio/mysql-sidecar:0.20.0
+        namespace: ${MYSQL_NAMESPACE}
         command:
-          - bash
-          - -o
-          - errexit
-          - -o
-          - pipefail
-          - -c
-          - |
-            s3_path="/mysql-backups/{{ .Deployment.Namespace }}/{{ .Deployment.Name }}/{{ toDate "2006-01-02T15:04:05.999999999Z07:00" .Time  | date "2006-01-02T15-04-05" }}/dump.sql.gz"
-            mysqldump -u root --password="${MYSQL_ROOT_PASSWORD}" --single-transaction --all-databases | gzip - | kando location push --profile '{{ toJson .Profile }}' --path ${s3_path} -
-  restore:
+        - bash
+        - -o
+        - errexit
+        - -o
+        - pipefail
+        - -c
+        - |
+          s3_path="/mysql-backups/${MYSQL_NAMESPACE}/${MYSQL_RELEASE}-mysql/{{ toDate "2006-01-02T15:04:05.999999999Z07:00" .Time  | date "2006-01-02T15-04-05" }}/dump.sql.gz"
+          mysqldump -u root --password="${MYSQL_ROOT_PASSWORD}" -h ${MYSQL_RELEASE}-mysql --single-transaction --all-databases | gzip - | kando location push --profile '{{ toJson .Profile }}' --path ${s3_path} -
+  delete:
```

Comment:
Also just a minor change request: although the order does not matter, just to maintain consistency across the blueprints, can we have actions as backup-restore-delete?

Reply:
Yes, I noticed that after raising the PR. I'll push those changes in the next revision. Thanks